pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -769,13 +769,21 @@ func (m *migrationMonitor) startMonitor() {
 
 // logMigrationInfo logs the same migration info as `virsh -r domjobinfo`
 func logMigrationInfo(logger *log.FilteredLogger, uid string, info *libvirt.DomainJobInfo) {
+	bToMB := func(bytes uint64) uint64 {
+		return bytes / 1000000
```

Comment:
Maybe better to:
```suggestion
		return bytes / (1024*1024)
```

Reply:
That'd be Mebibytes (MiB) instead of Megabytes (MB), but sure I can change it, MB are anyway seldom used nowadays.